servicediscovery/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,27 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package kubernetes
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/dapr/components-contrib/servicediscovery"
+)
+
+type Resolver struct {
+}
+
+func NewKubernetesResolver() *Resolver {
+	return &Resolver{}
+}
+
+func (z *Resolver) ResolveID(req *servicediscovery.ResolveRequest) (string, error) {
+	namespace := os.Getenv("NAMESPACE")
```

Comment:
I would define const for "NAMESPACE"

Reply:
I would get Namespace from req.Metadata, and have Dapr pass it in. Because its the Dapr runtime that is better off pulling environment variables and passing them to the component, which should expect to get it.

If Namespace is nil or empty, it should default to `default`.

Reply:
Done